target/linux/armvirt/Makefile

```diff
@@ -6,7 +6,7 @@ include $(TOPDIR)/rules.mk
 
 BOARD:=armvirt
 BOARDNAME:=QEMU ARM Virtual Machine
-FEATURES:=fpu pci rtc usb
+FEATURES:=fpu gpio pci rtc usb
```

Comment:
The gpio feature will automatically be set if the kernel configuration contains GPIO_SUPPORT, see: https://github.com/openwrt/openwrt/blob/master/scripts/target-metadata.pl#L23